### PR TITLE
26 Doors and Levels Game

### PR DESCRIPTION
**Pull request by Kouldred**
Created a game doors and levels. Setted (atleast_one_of_numers>=200) as condition to win the game.
#26

### CODE REVIEW

File: HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            {
+                Console.WriteLine("\n--------------------------");
+                Console.Write("Enter one of this numbers: ");
+                num.WriteNumbers();
+                string input = Console.ReadLine();
+                if (input == "")
+                {
+                    Console.WriteLine("\n!Invalid input!\n");
+                    continue;
+                } else i = Convert.ToInt32(input);
```
Comment (INIage) [inline HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs]: You don't check when user type some string and `Convert.ToInt32` throw exeption in this case

File: HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+using System.Linq;
+
+
+namespace NumbersGame
+{
+    class Program
+    {
+        struct NumbersOperator  //Stucture for storing our numbers and working with them
+        {
+            private int x0, x1, x2, x3, x4;
```
Comment (INIage) [inline HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs]: You use `Stack` but didn't use `List`. I belive you sould use some kind off array becouse if tomorow thay play with 20 numbers you will rewrite all youe code instad change one property :)

File: HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs (hunk 1 above)
Comment (Kouldred) [reply HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs]: I've added try/catch to handle this problem.

File: HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            
+            public void RandomNumbersGenerator()
```
Comment (KorolS) [inline HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs]: I think it's more suitable to use array in this case, and initiate it in loop.
The same considers the methods below CheckNumber, CheckWin: if value coincides with any first matching value in array, you can exit the loop, the same - comparing with given value.

File: HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs (hunk 2 above)
Comment (Kouldred) [reply HW1/Beliaev_Pavel/NumbersGame/NumbersGame/Program.cs]: I didn't use list or array, becouse i thought it was unnecessary for tactical soulution. I will change it in refactoring stage

**Pull request merged**